heroku_deploy/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine/git
+FROM node:alpine
```

Comment:
Needed due to `RUN curl https://cli-assets.heroku.com/install.sh | sh` fails to find "node".
![image](https://user-images.githubusercontent.com/55381256/82890968-51174880-9f4d-11ea-9487-c566d6acf94d.png)